md4c/md4c.h

```diff
@@ -268,6 +273,11 @@ typedef struct MD_SPAN_IMG_DETAIL {
     MD_ATTRIBUTE title;
 } MD_SPAN_IMG_DETAIL;
 
+/* Detailed info for MD_SPAN_WIKILINK. */
+typedef struct MD_SPAN_WIKILINK {
+    MD_ATTRIBUTE target;
+    MD_ATTRIBUTE text;
```

Comment:
The attribute `text` is IMHO not needed. It should rather be provided via `text()` callback as a normal text flow (and the way you update the renderer implementation suggests it already works that way).